package.json

```diff
@@ -67,7 +67,7 @@
   ],
   "scripts": {
     "pretest": "node scripts/copy_test_files.js",
-    "test": "nyc --reporter=html --reporter=text-summary mocha --timeout=10000 'dist/test/**/*.test.js'",
+    "test": "nyc --reporter=html --reporter=text-summary mocha --timeout=10000 'dist/test/**/*.test.js' --exclude 'dist/test/packages/**'",
```

Comment:
Because the `/packages/` folder contains a non-trivial amount of sample code we need to avoid accidentally picking up its own tests